cmd/telemeter-benchmark/main.go

```diff
@@ -120,14 +139,15 @@ func runCmd() error {
 		Interval:    opt.Interval,
 		MetricsFile: opt.MetricsFile,
 		Workers:     opt.Workers,
+		Logger:      opt.Logger,
 	}
 
 	b, err := benchmark.New(cfg)
 	if err != nil {
 		return fmt.Errorf("failed to configure the Telemeter benchmarking tool: %v", err)
 	}
 
-	log.Printf("Starting telemeter-benchmark against %s (listen=%s)", opt.To, opt.Listen)
+	level.Info(opt.Logger).Log("msg", fmt.Sprintf("Starting telemeter-benchmark against %s (listen=%s)", opt.To, opt.Listen))
```

Comment:
Using sprintf in combination with structured logging should be a red flag. :slightly_smiling_face: 

```suggestion
	level.Info(opt.Logger).Log("msg", "starting telemeter-benchmark", "to", opt.To, "addr", opt.Listen))
```